### PR TITLE
add version endpoint

### PR DESCRIPTION
**Pull request by RLuckom**
I was poking around the deployed environments and realized it's tricky to figure out what's deployed where. Rather than just try to stick this on someone's backlog I figured I'd just go for it. I hope you don't mind!

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/broadinstitute/leonardo.svg?branch=develop)](https://travis-ci.org/broadinstitute/leonardo) [![Coverage Status](https://coveralls.io/repos/github/broadinstitute/leonardo/badge.svg?branch=develop)](https://coveralls.io/github/broadinstitute/leonardo?branch=develop)
+[![Build Status](https://travis-ci.org/DataBiosphere/leonardo.svg?branch=develop)](https://travis-ci.org/DataBiosphere/leonardo) [![Coverage Status](https://coveralls.io/repos/github/DataBiosphere/leonardo/badge.svg?branch=develop)](https://coveralls.io/github/DataBiosphere/leonardo?branch=develop)
```
Comment (rtitle) [inline README.md]: Heh thanks

File: src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/StatusRoutesSpec.scala [hunk 2]
```diff
@@ -23,6 +23,14 @@ import scala.concurrent.duration._
 class StatusRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with CommonTestData with TestLeoRoutes with TestComponent {
   override implicit val patienceConfig = PatienceConfig(timeout = 1.second)
 
+  "GET /version" should "give 200 for ok" in {
+    eventually {
+      Get("/version") ~> leoRoutes.route ~> check {
+        status shouldEqual StatusCodes.OK
```
Comment (rtitle) [inline src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/StatusRoutesSpec.scala]: Could the test validate the actual version string returned? Does it return something for tests or `n/a`?

**Comment by rtitle** [comment]
A couple comments above.

A note: our git hooks to run automated tests, etc only work from branches in `DataBiosphere/leonardo`, not forks. So we typically ask people to make PRs from branches instead of forks (unless they're external).

**Comment by RLuckom** [comment]
Closing in favor of https://github.com/DataBiosphere/leonardo/pull/711 --when I tried to push the branch to this repo instead of my fork I was unable to open a PR

**Pull request closed without merging**